### PR TITLE
[ACS-7388][ADF] Break Toolbar dependency on Material Module

### PR DESCRIPTION
**Pull request by tamaragruszka**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [x] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://hyland.atlassian.net/browse/ACS-7388

**What is the new behaviour?**
component is standalone

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/src/lib/toolbar/toolbar.module.ts [hunk 1]
```diff
@@ -18,27 +18,13 @@
...
-        ToolbarComponent,
-        ToolbarTitleComponent,
-        ToolbarDividerComponent
-    ],
-    exports: [
-        ToolbarComponent,
-        ToolbarTitleComponent,
-        ToolbarDividerComponent
-    ]
+    imports: [CommonModule, ToolbarComponent, ToolbarTitleComponent, ToolbarDividerComponent, TranslateModule],
```
Comment (DenysVuika) [inline lib/core/src/lib/toolbar/toolbar.module.ts:27]: I think you don't need anymore: CommonModule, TranslateModule here

File: lib/core/src/lib/toolbar/toolbar.component.ts [hunk 2]
```diff
@@ -17,23 +17,24 @@
...
+import { MatToolbarModule } from '@angular/material/toolbar';
 
 @Component({
     selector: 'adf-toolbar',
+    standalone: true,
     templateUrl: './toolbar.component.html',
     styleUrls: ['./toolbar.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush,
     encapsulation: ViewEncapsulation.None,
+    imports: [MatToolbarModule],
```
Comment (DenysVuika) [inline lib/core/src/lib/toolbar/toolbar.component.ts:29]: also needs TranslateModule here

Comment (DenysVuika) [reply lib/core/src/lib/toolbar/toolbar.component.ts:29]: and CommonModule

Comment (tamaragruszka) [reply lib/core/src/lib/toolbar/toolbar.component.ts]: done

File: lib/core/src/lib/toolbar/toolbar.module.ts (hunk 1 above)
Comment (tamaragruszka) [reply lib/core/src/lib/toolbar/toolbar.module.ts]: done

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**